_docs/graphql-api.md

```diff
@@ -456,15 +456,16 @@ query event_by_tx {
 ```graphql
 {
   accountSnapshots(
-    first: 100,
+    first: 10,
     filter:{
       snapshotAtBlock: {
-        lessThan: "500"
+        greaterThan: "1500000",
```

Comment:
We usually want the latest result so I’m changing this to 1.5M instead of lessThan 500
Turing Staging tip: 1.9 M
Turing tip: 2.7M